rrfs-test/validated_yamls/templates/obtype_config/msonet_specificHumidity_188.yaml

```diff
@@ -14,7 +14,7 @@
          obsdataout:
            engine:
              type: H5File
-             obsfile: jdiag_msonet_specificHumidity_188.nc4
+             obsfile: jdiag_msonet_q188.nc4
```

Comment:
Should we change q188 to q_188?

Reply:
Do we really need to add an extra `_`? 
`jdiag_msonet_q188`
vs
`jdiag_msonet_q_188`